cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/AppLifecycleHttpHandler.java

```diff
@@ -183,9 +185,9 @@ public class AppLifecycleHttpHandler extends AbstractAppFabricHttpHandler {
   public BodyConsumer create(HttpRequest request, HttpResponder responder,
                              @PathParam("namespace-id") final String namespaceId,
                              @PathParam("app-id") final String appId)
-    throws BadRequestException, NamespaceNotFoundException, AccessException {
+          throws Exception {
```

Comment:
why are we changing this to generic `Exception`?

Reply:
that was an error. fixing it back